README.md

```diff
@@ -175,11 +175,20 @@ Include raw value in prop object as `prop['.rawValue']`
 **@param {boolean} [options.includeMeta]**
 Don't remove ".meta" key from a prop
 
+**@param {function} [options.jsonPreProcess]**
+A function that is ran before each YAML/JSON file is merged. Should return an object representing the modified JSON data.
+
 #### Example:
 
 ```js
 gulp.src('./design/props.json')
-  .pipe(theo.plugins.transform('web', { includeRawValue: true }))
+  .pipe(theo.plugins.transform('web', { 
+    includeRawValue: true,
+    jsonPreProcess: function (json) {
```

Comment:
Could we could use a fat arrow function? We're using ES6 throughout the examples so it might make sense to be consistent.

```js
jsonPreProcess: (json) => {
```